### DISCUSSION TITLE
Opened windows names and underline.

### DISCUSSION THREAD

**Discussion by fgh130** [valinet/ExplorerPatcher#1181] [General]
Is it possible to remove the names and underline opened windows in the TaskBar and replace them with a dot at the bottom under the icon?

**Comment by damian-tomczak** [comment]
![image](https://user-images.githubusercontent.com/43729842/227309027-d5ccf465-ff1f-4d4c-ba12-3dfacddd0063.png)

**Reply by fgh130** [reply]
Thx.
